ui-tests/src/test/resources/features/integrations-sf-db.feature

```diff
@@ -0,0 +1,91 @@
+@tp2
+@integrations-sf-db-test
+Feature: Test to verify correct function of connections kebab menu
+  https://app.zenhub.com/workspace/o/syndesisio/syndesis-qe/issues/102
+  https://drive.google.com/file/d/0B_udTBpEdqO8WUxzTWVKX1NsME0/view
+
+  @integrations-sf-db-clean-application-state
+  Scenario: Clean application state
+    Given "Camilla" logs into the Syndesis
+    Given clean application state
+
+  Scenario: Create salesforce connection
+      # create salesforce connection
+    When "Camilla" navigates to the "Connections" page
+    And clicks on the "Create Connection" button
+    And Camilla selects the "Salesforce" connection
+    Then she is presented with the "Validate" button
+      # fill salesforce connection details
+    When she fills "QE Salesforce" connection details
+    And scroll "top" "right"
+    And clicks on the "Next" button
+    And type "QE Salesforce" into connection name
+    And type "SyndesisQE salesforce test" into connection description
+    And clicks on the "Create" button
+    Then Camilla is presented with the Syndesis page "Connections"
+    # And she stays there for "3000" ms
+
+  @create-sf-db-integration
+  Scenario: Create integration from salesforce to postgresDB
+    When "Camilla" navigates to the "Home" page
+    And clicks on the "Create Integration" button to create a new integration.
+    Then she is presented with a visual integration editor
+    And she is prompted to select a "Start" connection from a list of available connections
+
+      # select salesforce connection as 'from' point
+    When Camilla selects the "QE Salesforce" connection
+    And she selects "On create" integration action
+    And she selects "Lead" from "sObjectName" dropdown
+    And clicks on the "Done" button
+
+      # select postgresDB connection as 'to' point
+    Then Camilla is presented with the Syndesis page "Choose a Finish Connection"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL stored procedure" integration action
+    And she selects "add_lead" from "procedureName" dropdown
+#    And she stays there for "13000" ms
+    And clicks on the "Done" button
+
+      # add data mapper step
+    And she stays there for "3000" ms
```

Comment:
Is this still needed? Could we wait for some event to happen rather?

Reply:
@dsimansk generally I'm using it, but here forgotten to comment out.